### PR TITLE
[docs] PR for Document Hasura in Ecosystem (Doc8338)

### PR DESCRIPTION
**Pull request by lizayugabyte**
Preview: https://deploy-preview-8381--infallible-bardeen-164bc9.netlify.app/latest/develop/ecosystem-integrations/graphql-hasura/

### CODE REVIEW

**Comment by lizayugabyte** [comment]
Comment via slack:
Nikhil:
Liza, current docs in Hasura is not out of date, only version of Hasura download needs to be updated to the latest.
New content should ideally be another page under Hasura
We can title the new page as - Build Linearly Scalable and Resilient Hasura Apps
Let’s talk tomorrow in the docs meeting where the content needs to go

File: docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+   accountname
+   givenname
+   familyname
+   city
+   user_orders(order_by: {orderid: desc}) {
+     orderid
+     ordertotal
+     orderdetails
+     ordertime
+     deliveryStatus
```
Comment (nchandrappa) [inline docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md:124]: We have to remove `deliveryStatus` attribute, currently, it's not being used.

File: docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+---
+
+You can use the [Hasura GraphQL Engine](https://hasura.io) with YugabyteDB to power your GraphQL applications with a distributed SQL database.
+
+For more information, see the following:
+
+- [Hasura GraphQL Engine documentation](https://docs.hasura.io)
+- [Scaling GraphQL Subscriptions to One Million Subscribers Linearly](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/1M_subscriptions.md)
+- [High Availability of GraphQL Subscriptions Under Infrastructure Failures](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/resilient_graphql_apps.md)
+- [Multi-Region GraphQL Applications](https://github.com/yugabyte/yugabyte-graphql-apps/blob/master/multi_region_apps.md)
```
Comment (nchandrappa) [inline docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md:22]: Let's remove `Multi-region GraphQL Applications`, it is still under development.

**Review by nchandrappa — COMMENTED** [review]
Liza, I've reviewed the doc content and it looks good. 

We've to figure out the right place to add this content though, increasingly it feels this content should be under the benchmarks page. We can discuss this further in the weekly docs meeting. Thanks

File: docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md:124]: Deleted

File: docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/develop/ecosystem-integrations/graphql-hasura.md:22]: Deleted

**Comment by lizayugabyte** [comment]
I'm going to add the doc to under Ecosystem for now, until it's decided how to restructure this section.

**Pull request merged**